Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -541,7 +544,7 @@ protected override Expression VisitRelinqUnknownNonExtension (Expression express
     /// <see cref="ExpressionVisitor"/> has a dedicated Visit method for it; otherwise, <see langword="false"/>. 
     /// Note that <see cref="ExpressionType.Extension"/>-type expressions are considered 'not supported' and will also return <see langword="false"/>.
     /// </returns>
-    private bool IsCurrentExpressionEvaluatable (Expression expression)
+    private static bool IsCurrentExpressionEvaluatable (Expression expression)
```

Comment:
Okay, great. Yes, please revert them. I agree about the explicit statelessness, but it's one of those code style things from way back that's spread throughout our entire codebase so I prefer the consistency.